### PR TITLE
feat: add border to media in media-block

### PR DESCRIPTION
**Pull request by niktverd**

### CODE REVIEW

**Review by gorgeousvlad — APPROVED** [review]

File: src/blocks/Media/Media.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ export const MediaBlock = (props: MediaBlockProps) => {
     return (
         <MediaBase {...props} onScroll={() => setPlay(true)}>
             <MediaBase.Card>
-                <Media {...mediaThemed} playVideo={play} />
+                <Media {...mediaThemed} playVideo={play} border={border} rounded />
```
Comment (yuberdysheva) [inline src/blocks/Media/Media.tsx:19]: What is `rounded`?

File: src/blocks/Media/__stories__/Media.mdx [hunk 2]
```diff
@@ -29,4 +29,6 @@ import * as MediaStories from './Media.stories.tsx';
 
 [`buttons?: Button[]` — An array with button objects](?path=/docs/documentation-types--docs#button)
 
+`border?: boolean` — Image border
```
Comment (yuberdysheva) [inline src/blocks/Media/__stories__/Media.mdx:32]: Can we use here types border: 'line' | 'shadow' | 'none'

we can set 'shadow' as !disableShadow and deprecate this prop

**Review by yuberdysheva — CHANGES_REQUESTED** [review]

File: src/components/Media/Media.scss [hunk 3]
```diff
@@ -13,4 +13,12 @@ $block: '.#{$ns}Media';
         display: flex;
         align-items: center;
     }
+
+    &_border {
+        border: 1px solid var(--g-color-line-generic);
+    }
+
+    &_rounded {
```
Comment (yuberdysheva) [inline src/components/Media/Media.scss:21]: It was rounded before if I am not mistaken?

**Review by yuberdysheva — CHANGES_REQUESTED** [review]

File: src/blocks/Media/__stories__/Media.mdx (hunk 2 above)
Comment (niktverd) [reply src/blocks/Media/__stories__/Media.mdx:32]: sure

File: src/blocks/Media/Media.tsx (hunk 1 above)
Comment (niktverd) [reply src/blocks/Media/Media.tsx]: PR was completely refactored. In discussion we came up with the solution that there should be an mixin added to media component

File: src/components/Media/Media.scss (hunk 3 above)
Comment (niktverd) [reply src/components/Media/Media.scss]: PR was completely refactored. In discussion we came up with the solution that there should be an mixin added to media component
This decision made this comment non-relevant

File: src/blocks/Media/Media.tsx [hunk 4]
```diff
@@ -4,19 +4,29 @@ import Media from '../../components/Media/Media';
...
+
+import './Media.scss';
+
+const b = block('media-block');
 
 export const MediaBlock = (props: MediaBlockProps) => {
-    const {media} = props;
+    const {media, border = 'shadow', disableShadow} = props;
+
+    const mediaBaseShadow = disableShadow || border !== 'shadow';
```
Comment (gorgeousvlad) [inline src/blocks/Media/Media.tsx:16]: We need another naming for `mediaBaseShadow` because it is boolean flag

**Review by gorgeousvlad — CHANGES_REQUESTED** [review]

File: src/blocks/Media/Media.tsx (hunk 4 above)
Comment (niktverd) [reply src/blocks/Media/Media.tsx]: changed

**Review by gorgeousvlad — APPROVED** [review]

**Pull request merged**